initfiles/sbin/complete-uninstall.sh.in

```diff
@@ -88,24 +135,30 @@ elif [ -e /etc/redhat-release -o -e /etc/SuSE-release ]; then
 fi
 
 echo "Removing Directory - ${path}"
-rm -rf ${path}
+removedir ${path}
+
+if [ -n "${mklink}" -a -n "${lpath}" ] ; then
```

Comment:
I agree this is not used often - but @AttilaVamos mentioned it will be used for testing.